backend/src/routes/comparisons/comparisons.router.ts

```diff
@@ -0,0 +1,14 @@
+import * as comparisonsController from '@controllers/comparisons/comparisons.controller';
+import { Router } from 'express';
+
+const PATH = '/comparisons';
+
+const comparisonsRouter = Router();
+
+comparisonsRouter.post(PATH, comparisonsController.addCarToComparison);
+comparisonsRouter.put(PATH, comparisonsController.changeComparisonType);
+comparisonsRouter.put(`${PATH}/clear`, comparisonsController.clearComparison);
```

Comment:
`PATCH` because we change only one prop of the instance. `PUT` is usually used we're able to change the whole instance. 

This separation isn't 100% strict, but let's stick to it to consistency